### PR TITLE
fix demois missing space

### PR DESCRIPTION
**Pull request by graingert**
This shows up as demois

Screenshot_20231201-181121

### CODE REVIEW

File: python/demo/main.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def api_index() -> list[AnyComponent]:
     # language=markdown
     markdown = """\
 This site providers a demo of [FastUI](https://github.com/samuelcolvin/FastUI), the code for the demo\
```
Comment (graingert) [inline python/demo/main.py:16]: Maybe just removing the \ at the end here will fix it. I think with markdown one newline becomes a single whitespace you need two new lines for a br

Comment (samuelcolvin) [reply python/demo/main.py:16]: ye, you're right - just removing the `\` fixes it.

Do you want to update your PR, or should I fix?

Comment (graingert) [inline python/demo/main.py:16]:
```suggestion
This site providers a demo of [FastUI](https://github.com/samuelcolvin/FastUI), the code for the demo
```

File: python/demo/main.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def api_index() -> list[AnyComponent]:
...
-is [here](https://github.com/samuelcolvin/FastUI/tree/main/python/demo).
+ is [here](https://github.com/samuelcolvin/FastUI/tree/main/python/demo).
```
Comment (graingert) [inline python/demo/main.py:17]:
```suggestion
is [here](https://github.com/samuelcolvin/FastUI/tree/main/python/demo).
```

**Comment by samuelcolvin** [comment]
Thanks so much. First external PR 👏 👏 👏 .

**Pull request merged**